redbot/cogs/audio/core/commands/controller.py

```diff
@@ -146,10 +150,12 @@ async def command_now(self, ctx: commands.Context):
 
         player.store("np_message", message)
 
-        dj_enabled = self._dj_status_cache.setdefault(
-            ctx.guild.id, await self.config.guild(ctx.guild).dj_enabled()
-        )
-        vote_enabled = await self.config.guild(ctx.guild).vote_enabled()
+        if ctx.guild.id in self._dj_status_cache:
+            dj_enabled = self._dj_status_cache[ctx.guild.id]
+        else:
+            dj_enabled = self._dj_status_cache.setdefault(ctx.guild.id, guild_data["dj_enabled"])
+
```

Comment:
These are unnecessary changes to the code logic, let's just keep it as similar to the original code as possible since this PR is meant for config optimizations, not moving code around (especially when the reason for this moving is unknown).
```suggestion
        dj_enabled = self._dj_status_cache.setdefault(ctx.guild.id, guild_data["dj_enabled"])
```